Mathlib/AlgebraicTopology/SimplexCategory/Defs.lean

```diff
@@ -167,6 +176,23 @@ noncomputable def inclusion.fullyFaithful (n : ℕ) :
 theorem Hom.ext {n} {a b : Truncated n} (f g : a ⟶ b) :
     f.toOrderHom = g.toOrderHom → f = g := SimplexCategory.Hom.ext _ _
 
+/-- Some quick attempts to prove that `⦋m⦌` is `n`-truncated (`⦋m⦌.len ≤ n`). -/
+macro "trunc" : tactic =>
```

Comment:
This should be `scoped` or even `local`, since `trunc` is too common a name to be reserved globally for simplex categories